### DISCUSSION TITLE
Solarerzeugung nur fürs Haus errechnen!?

### DISCUSSION THREAD

**Discussion by gickowtf** [gickowtf/enpal-homeassistant#2] [General]
Moin,
in der InfluxDB existiert leider nur `Energy.Production.Total.Day` aber nicht wie in der APP angezeigter Einzelwerte Solar fürs Haus

jetzt kann man sich aus `Energy.Production.Total.Day - Energy.Battery.Discharge.Day - Energy.External.Total.In.Day`
den gewünschten Wert errechnen. Nur leider stimmen diese Werte nicht zu 100% mit der APP überein. Kann es sein das dabei ein Faktor zu berücksichtigen ist ich meine irgendwo was gelesen zu haben das VA nicht gleich W sind ... bin leider kein Elektriker!

oder eben als Template 

`
{{ 
  states.sensor.enpal_production_day.state | float -
  states.sensor.enpal_energy_external_in_day.state | float -
  states.sensor.enpal_battery_discharge_day.state | float 
}}
`

**Comment by anticharm01** [comment]
https://www.thomas-krenn.com/de/wiki/Unterschied_zwischen_VA_(Voltampere)_und_Watt

PowerSensor: Power.Factor wäre der Mulitiplikator.
